app/screens/LoginScreen.tsx

```diff
@@ -35,72 +34,81 @@ export const LoginScreen = () => {
       .catch(() => alert("Wrong Email or Password"));
   };
 
+  const getThemeStyles = () => {
+    return {
+      container: {
+        flex: 1,
+        justifyContent: "center",
+        alignItems: "center",
+        backgroundColor: theme === "dark" ? "#1e1e1e" : "#fff",
+      },
+      title: {
+        fontSize: 24,
+        fontWeight: "bold",
+        marginBottom: 20,
+        color: theme === "dark" ? "#fff" : "#000",
+      },
+      input: {
+        width: "80%",
+        height: 40,
+        borderWidth: 1,
+        borderColor: theme === "dark" ? "#333" : "#ccc",
```

Comment:
It would be helpful for useTheme hook to have colours for selected theme to just use theme when defining styles
It may look like this:
```
const theme = useTheme()
const styles = createStyles(theme)

...


const createStyles = (theme: Theme) => StyleSheet.create({
   container: {
      ...
      backgroundColor: theme.colors.background
   }
})
```